ingest.go

```diff
@@ -2420,6 +2427,7 @@ func (d *DB) ingestApply(
 		}
 	}
 
+	// TODO(aaditya): should this metric also be incremented for flushableIngests?
```

Comment:
We count these under `Flush.AsIngest*` already which I guess covers this, but yeah it feels odd to not add to the ingestion count either. We should do this in the place where we increment the flush metric.

Reply:
Moving this to a separate PR since some of the metrics tests depend on these and I don't want to crowd this PR with that change.

Reply:
https://github.com/cockroachdb/pebble/pull/3429